Blog article on joint SCS standardization w/ ALASCA. (#790)
SovereignCloudStack/website: _i18n/en/_posts/blog/2023-11-23-joint-standardization.md

---
layout: post
title: "Joint Standardization by ALASCA and SCS"
author:
  - "Marius Feldmann"
  - "Kurt Garloff"
avatar:
  - "mfeldmann.jpg"
  - "kgarloff.jpg"
image: "blog/SCS-ALASCA.png"
about:
  - "garloff"
---

The Sovereign Cloud Stack (SCS) project develops open standards for the
provisioning of cloud and container infrastructure together with cloud
providers, users and partners. In addition, a complete, completely openly

> developed and production-ready softare implementation is offered together

Reviewer: ```suggestion
developed and production-ready software implementation is offered together
```

Author: Merged in PR #789

with broad knowledge on how to reliably operate it. Simplifying
the operations of modern cloud infrastructure facilitates the emergence
of a broad interoperable ecosystem of cloud offerings, supporting users
to achieve a higher level of digital sovereignty (or in other words
a higher level of self-determinism in the digital space). SCS is a project
of the Open Source Business Alliance e.V. (Federation for Digital Sovereignty)
and is funded by the BMWK (German Ministry for Economic Affairs and Climate
Action). SCS supports the realization of Gaia-X and the German Administrative
Cloud (DVC) and provides a technical implementation with a modular Open
Source solution.

After announcing the collaboration between SCS and ALASCA e.v. already
in January 2023 in a [joint statement](https://scs.community/en/2023/01/16/collaboration-of-alasca-and-scs/)
a lot of joint work has happened throughout the last months especially

> in the area of SCS standardization:

Reviewer: Warum der Doppelpunkt? Der nächste Absatz behandelt doch gar nicht die Vergangenheit.

Author: I think there was an additional sentence that said we would enhance the collaboration, which is why the colon would have made sense. I agree that it does no longer.

Starting today, the Special Interest Group for the standardization activities
of SCS will be governed by both the ALASCA project and the SCS project team

> and coordintated by Dr. Matthias Büchse from Cloud & Heat technologies.

Reviewer: ```suggestion
and coordintated by Dr. Matthias Büchse from Cloud & Heat Technologies.
```

Author: Merged in PR #789

Building on top of already existing standards, the goal is to achieve a
broader adoption of SCS standards by more close collaboration with ALASCA

> and also cover the ALASCA technology stack with the cloud lifecycle management

Reviewer: ```suggestion
and also cover the ALASCA technology stack with the cloud-lifecycle management
```
(weil cloud sich auf lifecycle und nicht auf management tool bezieht)

Author: I disagree

> tool yaook. Providing a second standards-compliant implementation of SCS

Reviewer: ```suggestion
tool Yaook. Providing a second standards-compliant implementation of SCS
```

> standards on top of yaook validates the universal nature of the standards.

Reviewer: ```suggestion
standards on top of Yaook validates the universal nature of the standards.
```

The broader setup also supports the high quality of the evolving standards
and the sustainability of the SCS standardization efforts. The Special

> Interest Group is open to all cloud providers and integrator that want

Reviewer: ```suggestion
Interest Group is open to all cloud providers and integrators that want
```

Author: Merged in PR #789

to contribute and provide SCS-compliant services.

> For ALASCA, the closer collaboration in the evolution of the standards
> als offers significant value: The association strives enabling digitally
> sovereign setup and operations of own cloud infrastructure by research,
> educational or generally public entities as well as private sector
> companies and individuals based on Open Source Software and the connected
> supply of competencies. With the SCS standards, this effort receives
> a unique foundation, upgrading the relevance of the ALASCA driven
> Open Source Software. The ALASCA community significantly benefits from
> the additional support by the SCS community.

Reviewer: ```suggestion
Likewise for ALASCA, the closer collaboration in the evolution of the standards
offers significant value: The association pursues enabling digitally
sovereign setups and operations of own cloud infrastructure by research,
educational or generally public entities as well as private sector
companies and individuals based on Open-Source Software and the connected
supply of competencies. With the SCS standards, this effort receives
a unique foundation, upgrading the relevance of the ALASCA-driven
Open-Source Software. The ALASCA community significantly benefits from
the additional support by the SCS community.
```

Author: Partially addressed.
